opendatahub/quickstart/basic/README.md

```diff
@@ -51,7 +51,7 @@ spec:
 
 Deploy the sample model:
 ~~~
-./deploy.sh
+./opendatahub/quickstart/basic/deploy.sh
```

Comment:
each folder has to have deploy.sh file (https://github.com/opendatahub-io/modelmesh-serving/tree/main/opendatahub/quickstart) 
and the one of pre-requisite docs explains how to use the script.
https://github.com/opendatahub-io/modelmesh-serving/blob/main/opendatahub/quickstart/common_docs/modelmesh-install.md

~~~
source ../env.sh

cd opendatahub/quickstart/$QUICK_START_FOLDER # (for example, cd opendatahub/quickstart/basic)

./deploy.sh
~~~
So I don't think this is needed.